p2p/transport/webrtc/util.go

```diff
@@ -0,0 +1,77 @@
+package libp2pwebrtc
```

Comment:
in general this file contains a lot of intensive string operations. And while modern hardware is great and Go's GC is awesome, it still feels to me that it might be worth trying to reduce many of the allocs here?!

Reply:
Fixed and benchmarks shared.